# Genome-wide association analysis

## Genotype calling algorithms

## DNA processing quality control

## Sample quality control

### Cryptic relatedness

### Population stratification

### Heterozygosity and missingness outliers

### Differential missingness

### Sex chromosome anomalies

## Marker quality control

### Genotyping concordance

### Switch rate

### Genotype call rate

### Minor allele frequency

### Hardy-Weinberg equilibrium outliers

### Additional QC for regions like MHC

### Ambigious nucleotides

### Non-matching nucleotides

## X-chromosome quality control

## Single marker regression

### Trend test

### Alleles test

## Two-stage approach

## Haplotype GWAs design

### Genomic control

## Gene-based GWAS

## Gene-set GWAS

## Extensions to binary and categorical phenotypes

### Threshold model

## Analysis of rare variants

## Analysis of X, Y and mitochondrial chromosomes

## Analysis of copy number variants

### Common variation

### Rare variation

Check Lee et al (2014) for a review

Due to the low frequencies of rare variants, classical single marker tests commonly used in genome-wide association studies (GWAS) for studying common variants effects are not applicable.

In view of the lack of power of single marker analysis of rare variants, methods investigating rare variation are typically region-based tests where one tests for the cumulative effects of the rare variants in a region. These region-based methods can be broadly classified into three classes: burden tests, non-burden tests and hybrid of the two. The key difference between burden and non-burden tests is how the cumulative effects of the rare variants are combined for association testing. For the commonly used simple burden tests, one summarizes the rare variants within a region as a single summary genetic burden variable, e.g. the total number of rare variants in a region, and tests its association with a trait. Burden tests implicitly assume all the rare variants in the region under consideration are causal and are associated with the phenotype in the same direction and magnitude. Hence, they all share the limitation of substantial power loss when there are many non-causal genetic variants in a region and/or when there are both protective and harmful variants.

Several region-based non-burden tests have been proposed by aggregating marginal test statistics (Neale et al., 2011; Basu and Pan, 2011; Lin and Tang, 2011). One such test is the sequence kernel association test (SKAT) (Wu et al., 2011), where one summarizes the rare variants in the region using a kernel function, and then test for association with the trait of interest using a variance component score test. SKAT is robust to the signs and magnitudes of the associations of rare variants with a trait. It is more powerful than the burden tests when the effects are in different directions or the majority of variants in a region are null, but is less powerful than burden tests when most variants in a region are causal and the effects are in the same direction. Several hybrids of the two methods have been proposed to improve test power and robustness (Lee et al., 2012; Derkach et al., 2013; Sun et al., 2013).

## Analysis of multi-ethnic samples

## Analysis of indirect genetic effects

## GWAS vs whole-genome association

## Analysis of multiple traits

## Mixed-model association analysis

## Penalized regression GWAS

## Bayesian GWAS

## Machine learning for GWAS

