# Gene-environment interaction

Identification of gene-environment interactions has important implications for understanding underlying disease etiology and developing disease prevention and intervention strategies.

## Single step methods

### Case-control

### Case-only

### Empirical Bayes and Bayesian Model Averaging

## Multi stage methods

## Joint tests

## Gene-based interaction tests

There are multiple reasons for using set-based gene-environment interaction tests. 

1. Multiple comparison adjustments for a large number of markers across the genome could result in power loss.
2. Closely located SNPs are correlated because of linkage disequilibrium. Multiple tests for GxE in these single-marker-based GxE models are even more dependent, as interaction terms in these models share the same environmental variable. Dependence among multiple tests can result in incorrect Type 1 error rates and causes bias in standard multiple comparison adjustments and this bias is often difficult to correct.
3. The single-marker GxE test does not interrogate the joint effects of multiple SNPs that have similar biological functions. When the main effects of multiple SNPs in a set are  associated with a disease/trait, the classical single marker regression interaction test can be biased.

Lin et al. (2013) developed a method to analyse GxE for a set of markers using generalized linear models. The method tests for SNP-set by environment interactions using a variance component test, and because a set of variants will likely be correlated, the main SNP effect estimates under the null hypothesis are obtained using ridge regression. The software is called GESAT. Here, they model GxE effects as random.

## Combining multiple environments

## Variance heterogeneity

### Levene's test

### Two-step screening on residual variance heterogeneity

## Conditional quantile regression 

## RELIEF and other machine learning tools

### Multidimensionality reduction

## Meta-analytic GxE approaches
