--- 
title: "A handbook for Computational Genetics"
author: "Alfred Pozarickij"
date: "`r Sys.Date()`"
site: bookdown::bookdown_site
output: bookdown::gitbook
documentclass: book
bibliography: [book.bib, packages.bib]
biblio-style: apalike
link-citations: yes
github-repo: rstudio/bookdown-dem
description: "This is the book describing theoretical and practical approaches in analysis of genome data."
---

# Preface {-}

The scope of this book is to provide an outline of computational methods available for the analysis of genetic data.

First chapter introduces methods to infer population parameters. 

Next X chapters focus on population genetics.

In this book, the amount of mathematics and statistics is kept to a minimum. Only methods designed specifically to address issues in genetics are shown. I have produced another book [insert link here], which is intended to familiarise the reader with commonly used approaches and develop some intuition behind them. By no means the list is comprehensive and only serves as a quick guide. The internet provides much more information regarding this topic.

Rather than providing references at the end of each chapter, I decided to combine them into supplementary text [insert link here]. References are arranged according to different topics discussed in this book. I tried to do my best to cause as little confusion as possible.

Finally, don't hesitate to contact me if I have not included your favorite method (apozarickij@gmail.com). I would be more than happy to hear about it.








<!--chapter:end:index.Rmd-->

# (PART) Quantitative Genetics {-}

# Population parameters

## Mean

## Variance

## Covariance

## Genetic correlation

## Additivity

## Dominance/Recesivness

## Codominance

<!--chapter:end:01-Descriptives.Rmd-->

# Sequencing technologies

<!--chapter:end:02-Sequencing-technologies.Rmd-->

# Genome-wide association analysis

## Genotype calling algorithms

## DNA processing quality control

## Sample quality control

### Cryptic relatedness

### Population stratification

### Heterozygosity and missingness outliers

### Differential missingness

### Sex chromosome anomalies

## Marker quality control

### Genotyping concordance

### Switch rate

### Genotype call rate

### Minor allele frequency

### Hardy-Weinberg equilibrium outliers

### Additional QC for regions like MHC

## X-chromosome quality control

## Single marker regression

### Trend test

### Alleles test

## Two-stage approach

## Haplotype GWAs design

### Genomic control

## Gene-based GWAS

## Gene-set GWAS

## Extensions to binary and categorical phenotypes

### Threshold model

## Analysis of rare variants

## Analysis of X, Y and mitochondrial chromosomes

## Analysis of copy number variants

### Common variation

### Rare variation

## Analysis of multi-ethnic samples

## Analysis of indirect genetic effects

## GWAS vs whole-genome association

## Analysis of multiple traits

## Mixed-model association analysis

## Penalized regression GWAS

## Bayesian GWAS

## Machine learning for GWAS


<!--chapter:end:03-Genome-wide-association-analyses.Rmd-->

# Heritability

## Realized heritability

### Evolvability

### Reliability

## Twin studies

## GCTA

## LD-score regression

## LDAK

<!--chapter:end:04-Heritability.Rmd-->

# Genomic prediction

## Polygenic risk scores

## Gene-based polygenic score (POLARIS)

## Pathway-based polygenic risk score

## LD adjusted PRS

## BLUP

## Bayesian Zoo

## Reproducing kernel Hilbert space

## Machine learning methods


<!--chapter:end:05-Genomic-prediction.Rmd-->

# Pleiotropy

## Direct

## Indirect


<!--chapter:end:06-Pleiotropy.Rmd-->

# Pathway-analysis

<!--chapter:end:07-Pathway-analysis.Rmd-->

# Functional annotation

<!--chapter:end:08-Functional-annotation.Rmd-->

# Causal inference

## Gene-knockout

## Conditioning

## Finemapping

## Mendelian Randomization

<!--chapter:end:09-Inferring-causality.Rmd-->

# Combining multiple datasets

## Meta-analysis

## Mega-analysis

<!--chapter:end:10-Analysis-of-multiple-datasets.Rmd-->

# Gene-environment interaction

Identification of gene-environment interactions has important implications for understanding underlying disease etiology and developing disease prevention and intervention strategies.

## Single step methods

### Case-control

### Case-only

### Empirical Bayes and Bayesian Model Averaging

## Multi stage methods

## Joint tests

## Set-based interaction tests

There are multiple reasons for using set-based gene-environment interaction tests. 

1. Multiple comparison adjustments for a large number of markers across the genome could result in power loss.
2. Closely located SNPs are correlated because of linkage disequilibrium. Multiple tests for GxE in these single-marker-based GxE models are even more dependent, as interaction terms in these models share the same environmental variable. Dependence among multiple tests can result in incorrect Type 1 error rates and causes bias in standard multiple comparison adjustments and this bias is often difficult to correct.
3. The single-marker GxE test does not interrogate the joint effects of multiple SNPs that have similar biological functions. When the main effects of multiple SNPs in a set are  associated with a disease/trait, the classical single marker regression interaction test can be biased.

Lin et al. (2013) developed a method to analyse GxE for a set of markers using generalized linear mixed models. The method tests for SNP-set by environment interactions using a variance component test, and because a set of variants will likely be correlated, the main SNP effect estimates under the null hypothesis are obtained using ridge regression. 
Their software is called GESAT. Here, they model GxE effects as random, as opposed to the  classical approach of treating *βj*’s as fixed effects followed by a test with *p* degrees of freedom. The latter approach can suffer from power loss when *p* is moderate/large, and numerical difficulties when some genetic markers in the set are in high LD. 
The model allows to adjust for the main effects of all SNPs while simultaneously testing for the interactions between the SNPs in the region and environmental variable. For unbalanced designs when a binary environmental exposure has a low frequency in one category, GESAT is most advantageous over single marker regression GxE test. Such unbalanced designs can occur due to case–control sampling and the strong association of an environmental factor with disease. When the effect size is modest, GESAT performs better that single marker regression GxE test, but when the effect size is strong, the opposite is true. 
Their simulations suggest that the power of GESAT seems fairly robust to the dependence between G and E.

The same approach can be applied to investigating various other biological problems. For example, we can test for the interactions between gene expressions in a pathway or network and an environmental variable by simply replacing G by gene expressions in a gene-set.

## Combining multiple environments

## Variance heterogeneity

### Levene's test

### Two-step screening on residual variance heterogeneity

## Conditional quantile regression 

## RELIEF and other machine learning tools

### Multidimensionality reduction

## Meta-analytic GxE approaches

<!--chapter:end:11-GxE.Rmd-->

# Gene-gene interaction

## Single step methods

## Multi stage methods

## Machine learning methods

<!--chapter:end:12-GxG.Rmd-->

# Other omics

## Transcriptome-wide association studies

### cis eQTLs

### trans eQTLs

### 3-D structure of the genome

## Phenome-wide association studies

## Metabolomics

## Epigenomics

<!--chapter:end:13-Multi-omics.Rmd-->

# Quantitative trait loci mapping



<!--chapter:end:14-QTL-mapping.Rmd-->

# Additional point to consider

## Kinship matrix

### Path coefficients

## Genetic relationship matrix

## Animal models

## Phasing

## Haplotyping

## Statistical power

### When marker is a disease susceptibility locus

### When marker is not disease susceptability locus

## Multiple comparisons

### Effective number of independant variants

## Biases

### Ascertainment

### Attenuation

### Selection

## Family studies

### Transmission disequilibrium tests

## Twin studies

## Adoption studies

## Equifinality (many genes give same trait)

## Gene dosage

### Allelic dosage

## Allelic heterogeneity

## Genetic heterogeneity

## Genomic imprinting

## Penetrance/phenocopy

## Endophenotypes

## Ploidy

## Exended phenotype

## Genome sizes








<!--chapter:end:15-Extra.Rmd-->

# (PART) Population Genetics {-}

# Genetic drift

<!--chapter:end:16-Genetic-drift.Rmd-->

# Mutation


## Mutation age

<!--chapter:end:17-Mutation.Rmd-->

# Selection

## Directional

## Balancing

### Frequency-dependent selection 1

### Frequency dependent selection 2


## Background selection

<!--chapter:end:18-Selection.Rmd-->

# Migration

<!--chapter:end:19-Migration.Rmd-->

# Diversity

<!--chapter:end:20-Diversity.Rmd-->

# Admixture

<!--chapter:end:21-Admixture.Rmd-->

# Linkage disequilibrium

<!--chapter:end:22-LD.Rmd-->

# In breeding and heterosis

<!--chapter:end:23-Inbreeding-and-heterosis.Rmd-->

# Assortative mating

<!--chapter:end:24-Assortative-mating.Rmd-->

# Identity

## IBS

### Long runs of IBT

## IBD

## IBT


<!--chapter:end:25-Identity.Rmd-->

# Neutral theory of molecular evolution

## Nearly neutral theory of molecular evolution

<!--chapter:end:26-Neutral-theory.Rmd-->

