\documentclass[]{book}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={A handbook for Computational Genetics},
            pdfauthor={Alfred Pozarickij},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{longtable,booktabs}
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{A handbook for Computational Genetics}
  \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
  \author{Alfred Pozarickij}
  \preauthor{\centering\large\emph}
  \postauthor{\par}
  \predate{\centering\large\emph}
  \postdate{\par}
  \date{2018-07-28}


\usepackage{amsthm}
\newtheorem{theorem}{Theorem}[chapter]
\newtheorem{lemma}{Lemma}[chapter]
\theoremstyle{definition}
\newtheorem{definition}{Definition}[chapter]
\newtheorem{corollary}{Corollary}[chapter]
\newtheorem{proposition}{Proposition}[chapter]
\theoremstyle{definition}
\newtheorem{example}{Example}[chapter]
\theoremstyle{definition}
\newtheorem{exercise}{Exercise}[chapter]
\theoremstyle{remark}
\newtheorem*{remark}{Remark}
\newtheorem*{solution}{Solution}
\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\chapter*{Preface}\label{preface}
\addcontentsline{toc}{chapter}{Preface}

The scope of this book is to provide an outline of computational methods
available for the analysis of genetic data.

First chapter introduces methods to infer population parameters.

Next X chapters focus on population genetics.

In this book, the amount of mathematics and statistics is kept to a
minimum. Only methods designed specifically to address issues in
genetics are shown. I have produced another book {[}insert link here{]},
which is intended to familiarise the reader with commonly used
approaches and develop some intuition behind them. By no means the list
is comprehensive and only serves as a quick guide. The internet provides
much more information regarding this topic.

Rather than providing references at the end of each chapter, I decided
to combine them into supplementary text {[}insert link here{]}.
References are arranged according to different topics discussed in this
book. I tried to do my best to cause as little confusion as possible.

Finally, don't hesitate to contact me if I have not included your
favorite method
(\href{mailto:apozarickij@gmail.com}{\nolinkurl{apozarickij@gmail.com}}).
I would be more than happy to hear about it.

\part{Quantitative
Genetics}\label{part-quantitative-genetics}

\chapter{Population parameters}\label{population-parameters}

\section{Mean}\label{mean}

\section{Variance}\label{variance}

\section{Covariance}\label{covariance}

\section{Genetic correlation}\label{genetic-correlation}

\section{Additivity}\label{additivity}

\section{Dominance/Recesivness}\label{dominancerecesivness}

\section{Codominance}\label{codominance}

\chapter{Sequencing technologies}\label{sequencing-technologies}

\chapter{Genome-wide association
analysis}\label{genome-wide-association-analysis}

\section{Genotype calling algorithms}\label{genotype-calling-algorithms}

\section{DNA processing quality
control}\label{dna-processing-quality-control}

\section{Sample quality control}\label{sample-quality-control}

\subsection{Cryptic relatedness}\label{cryptic-relatedness}

\subsection{Population stratification}\label{population-stratification}

\subsection{Heterozygosity and missingness
outliers}\label{heterozygosity-and-missingness-outliers}

\subsection{Differential missingness}\label{differential-missingness}

\subsection{Sex chromosome anomalies}\label{sex-chromosome-anomalies}

\section{Marker quality control}\label{marker-quality-control}

\subsection{Genotyping concordance}\label{genotyping-concordance}

\subsection{Switch rate}\label{switch-rate}

\subsection{Genotype call rate}\label{genotype-call-rate}

\subsection{Minor allele frequency}\label{minor-allele-frequency}

\subsection{Hardy-Weinberg equilibrium
outliers}\label{hardy-weinberg-equilibrium-outliers}

\subsection{Additional QC for regions like
MHC}\label{additional-qc-for-regions-like-mhc}

\subsection{Ambigious nucleotides}\label{ambigious-nucleotides}

\subsection{Non-matching nucleotides}\label{non-matching-nucleotides}

\section{X-chromosome quality
control}\label{x-chromosome-quality-control}

\section{Single marker regression}\label{single-marker-regression}

\subsection{Trend test}\label{trend-test}

\subsection{Alleles test}\label{alleles-test}

\section{Two-stage approach}\label{two-stage-approach}

\section{Haplotype GWAs design}\label{haplotype-gwas-design}

\subsection{Genomic control}\label{genomic-control}

\section{Multimarker gene-set based
approaches}\label{multimarker-gene-set-based-approaches}

\url{https://cran.r-project.org/web/packages/aSPU/aSPU.pdf}

\section{Extensions to binary and categorical
phenotypes}\label{extensions-to-binary-and-categorical-phenotypes}

\subsection{Threshold model}\label{threshold-model}

\section{Analysis of rare variants}\label{analysis-of-rare-variants}

Check Lee et al (2014) for a review

Due to the low frequencies of rare variants, classical single marker
tests commonly used in genome-wide association studies (GWAS) for
studying common variants effects are not applicable. In view of the lack
of power of single marker analysis of rare variants, methods
investigating rare variation are typically region-based tests where one
tests for the cumulative effects of the rare variants in a region. These
region-based methods can be broadly classified into three classes:
burden tests, non-burden tests and hybrid of the two. The key difference
between burden and non-burden tests is how the cumulative effects of the
rare variants are combined for association testing. For the commonly
used simple burden tests, one summarizes the rare variants within a
region as a single summary genetic burden variable, e.g.~the total
number of rare variants in a region, and tests its association with a
trait. Burden tests implicitly assume all the rare variants in the
region under consideration are causal and are associated with the
phenotype in the same direction and magnitude. Hence, they all share the
limitation of substantial power loss when there are many non-causal
genetic variants in a region and/or when there are both protective and
harmful variants. Several region-based non-burden tests have been
proposed by aggregating marginal test statistics (Neale et al., 2011;
Basu and Pan, 2011; Lin and Tang, 2011). One such test is the sequence
kernel association test (SKAT) (Wu et al., 2011), where one summarizes
the rare variants in the region using a kernel function, and then test
for association with the trait of interest using a variance component
score test. SKAT is robust to the signs and magnitudes of the
associations of rare variants with a trait. It is more powerful than the
burden tests when the effects are in different directions or the
majority of variants in a region are null, but is less powerful than
burden tests when most variants in a region are causal and the effects
are in the same direction. Several hybrids of the two methods have been
proposed to improve test power and robustness (Lee et al., 2012; Derkach
et al., 2013; Sun et al., 2013).

\subsection{Collapsing methods based on pooling multiple rare
variants}\label{collapsing-methods-based-on-pooling-multiple-rare-variants}

\subsubsection{Sum test}\label{sum-test}

The most powerful multi-marker test when there are no causal variants
with effects in opposite directions and when there are few or no
non-causal RVs. Otherwise, it suffers from substantial loss of power.

\subsubsection{Cohort Allelic Sums test
(CAST)}\label{cohort-allelic-sums-test-cast}

\subsubsection{Combined Multivariate Collapsing
(CMC)}\label{combined-multivariate-collapsing-cmc}

\subsubsection{Weighted Sum test (W-Sum)}\label{weighted-sum-test-w-sum}

\subsubsection{Kernel Based Adaptive Cluster
(KBAC)}\label{kernel-based-adaptive-cluster-kbac}

\subsubsection{Replication Based Test
(RBT)}\label{replication-based-test-rbt}

\subsubsection{ARIEL test}\label{ariel-test}

\subsubsection{The EREC test}\label{the-erec-test}

\subsection{Methods treating rare variant effects as
random}\label{methods-treating-rare-variant-effects-as-random}

\subsubsection{The SSU approach}\label{the-ssu-approach}

Has good power in the presence of opposite association directions and
non-causal RVs.

\subsubsection{C-alpha test}\label{c-alpha-test}

\subsubsection{SKAT}\label{skat}

Has good power in the presence of opposite association directions and
non-causal RVs. It was recently suggested that using SKAT in the
presence of RVs and common variants (CVs) may be less optimal because
RVs are weighted to have much more importance than CVs (Ionita-Laza et
al., 2013).

\subsection{Methods based on model
selection}\label{methods-based-on-model-selection}

The model-selection approaches perform in the middle of random eﬀect and
collapsing methods. One issue common to model-selection methods is that
model selection approaches use dimension-reduction strategies to
substantially reduce the number of parameters one would require to ﬁt
these large number of RVs. Hence, any model we can construct will never
be the true model that generated the data we observe. In other words,
the set of models is clearly misspeciﬁed, and model selection is best
seen as a way of approximating, rather than identifying, full reality
(Burnham and Anderson (2002), pp.~20-23).

\subsubsection{Seq-aSum}\label{seq-asum}

\subsubsection{Seq-aSum-VS}\label{seq-asum-vs}

The Seq-aSum-VS approach classiﬁes RVs based on the direction of
association (`+1' for positive association, `-1' for negative
association and `0' for no association) and implements a sequential
variable selection scheme to select the best model for association
between the SNP-set and the disease. The only diﬀerence between the
Seq-aSum approach and the Seq-aSum-VS approach is that the variable
selection (`0' allocation for a variant) is not implemented in the
former. The Seq-aSum-VS approach starts with putting all the RVs in the
`+1' group and proceeds by moving each RV sequentially to the other two
groups and ﬁnally chooses the allocation (`+1',`-1', or `0' ) with
highest likelihood to the RV. The process of choosing the best model in
Basu and Pan (2011)'s method can be compared to a stepwise regression,
where one may not always ﬁnd the best model due to this selection
scheme. This is especially true if a particular allocation results in a
slightly higher likelihood than the other two allocations. In this case,
choosing the allocation with highest likelihood for a SNP might not be
optimal, rather it might be more eﬃcient to allow multiple allocations
for a RV and construct a test that takes into account multiple plausible
models for the disease-RV association. Moreover, the performance of the
sequential search often depends on the ordering of the variants in this
search mechanism. A model-averaging approach could potentially reduce
the dependency on the ordering of the variants in this sequential
search.

\subsubsection{Variable Threshold Test
(VT)}\label{variable-threshold-test-vt}

\subsubsection{RARECOVER}\label{rarecover}

\subsubsection{Selective grouping
method}\label{selective-grouping-method}

\subsubsection{Step-Up}\label{step-up}

\subsection{Combination of collapsing and random effects
methods}\label{combination-of-collapsing-and-random-effects-methods}

According to Basu and Pan (2011), the model selection methods,
especially Seq-aSum-VS approach, performed very well when there were
both protective and deleterious causal RVs and very few non-causal RVs,
but the performance of the Seq-aSum-VS approach was not very impressive
in the presence of a moderate or large number of non-causal RVs. These
and other ﬁndings (Basu and Pan, 2011) have led to combining the
strengths of collapsing and random eﬀect methods.

\subsubsection{SKAT-O}\label{skat-o}

\subsubsection{SKAT-C}\label{skat-c}

\subsubsection{Fisher method}\label{fisher-method}

\subsubsection{MiST}\label{mist}

\section{Analysis of X, Y and mitochondrial
chromosomes}\label{analysis-of-x-y-and-mitochondrial-chromosomes}

\section{Analysis of copy number
variants}\label{analysis-of-copy-number-variants}

\subsection{Common variation}\label{common-variation}

\subsection{Analysis of rare
variants}\label{analysis-of-rare-variants-1}

\section{Analysis of multi-ethnic
samples}\label{analysis-of-multi-ethnic-samples}

\section{Analysis of indirect genetic
effects}\label{analysis-of-indirect-genetic-effects}

\section{GWAS vs whole-genome
association}\label{gwas-vs-whole-genome-association}

\section{Analysis of multiple traits}\label{analysis-of-multiple-traits}

\section{Mixed-model association
analysis}\label{mixed-model-association-analysis}

\section{Penalized regression GWAS}\label{penalized-regression-gwas}

\section{Bayesian GWAS}\label{bayesian-gwas}

\section{Machine learning for GWAS}\label{machine-learning-for-gwas}

\chapter{Heritability}\label{heritability}

\section{Realized heritability}\label{realized-heritability}

\subsection{Evolvability}\label{evolvability}

\subsection{Reliability}\label{reliability}

\section{Twin studies}\label{twin-studies}

\section{GCTA}\label{gcta}

\section{LD-score regression}\label{ld-score-regression}

\section{LDAK}\label{ldak}

\chapter{Genomic prediction}\label{genomic-prediction}

\section{Polygenic risk scores}\label{polygenic-risk-scores}

\section{Gene-based polygenic score
(POLARIS)}\label{gene-based-polygenic-score-polaris}

\section{Pathway-based polygenic risk
score}\label{pathway-based-polygenic-risk-score}

\section{LD adjusted PRS}\label{ld-adjusted-prs}

\subsection{LDpred with functional
annotation}\label{ldpred-with-functional-annotation}

\section{Annopred}\label{annopred}

\section{Pleiopred}\label{pleiopred}

\section{BLUP}\label{blup}

\section{Bayesian Zoo}\label{bayesian-zoo}

\section{Reproducing kernel Hilbert
space}\label{reproducing-kernel-hilbert-space}

\section{Machine learning methods}\label{machine-learning-methods}

\chapter{Pleiotropy}\label{pleiotropy}

\section{Direct}\label{direct}

\section{Indirect}\label{indirect}

\chapter{Pathway-analysis}\label{pathway-analysis}

\chapter{Functional annotation}\label{functional-annotation}

\chapter{Causal inference}\label{causal-inference}

\section{Gene-knockout}\label{gene-knockout}

\section{Conditioning}\label{conditioning}

\section{Finemapping}\label{finemapping}

\section{Mendelian Randomization}\label{mendelian-randomization}

\chapter{Combining multiple datasets}\label{combining-multiple-datasets}

\section{Meta-analysis}\label{meta-analysis}

\section{Mega-analysis}\label{mega-analysis}

\chapter{Gene-environment
interaction}\label{gene-environment-interaction}

Identification of gene-environment interactions has important
implications for understanding underlying disease etiology and
developing disease prevention and intervention strategies.

\section{Single step methods}\label{single-step-methods}

\subsection{Case-control}\label{case-control}

\subsection{Case-only}\label{case-only}

\subsection{Empirical Bayes and Bayesian Model
Averaging}\label{empirical-bayes-and-bayesian-model-averaging}

\section{Multi stage methods}\label{multi-stage-methods}

\section{Joint tests}\label{joint-tests}

\section{Set-based interaction tests}\label{set-based-interaction-tests}

There are multiple reasons for using set-based gene-environment
interaction tests.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Multiple comparison adjustments for a large number of markers across
  the genome could result in power loss.
\item
  Closely located SNPs are correlated because of linkage disequilibrium.
  Multiple tests for GxE in these single-marker-based GxE models are
  even more dependent, as interaction terms in these models share the
  same environmental variable. Dependence among multiple tests can
  result in incorrect Type 1 error rates and causes bias in standard
  multiple comparison adjustments and this bias is often difficult to
  correct.
\item
  The single-marker GxE test does not interrogate the joint effects of
  multiple SNPs that have similar biological functions. When the main
  effects of multiple SNPs in a set are associated with a disease/trait,
  the classical single marker regression interaction test can be biased.
\end{enumerate}

Lin et al. (2013) developed a method to analyse GxE for a set of markers
using generalized linear mixed models. The method tests for SNP-set by
environment interactions using a variance component test, and because a
set of variants will likely be correlated, the main SNP effect estimates
under the null hypothesis are obtained using ridge regression. Their
software is called GESAT. Here, they model GxE effects as random, as
opposed to the classical approach of treating \emph{BETAj}'s as fixed
effects followed by a test with \emph{p} degrees of freedom. The latter
approach can suffer from power loss when \emph{p} is moderate/large, and
numerical difficulties when some genetic markers in the set are in high
LD. The model allows to adjust for the main effects of all SNPs while
simultaneously testing for the interactions between the SNPs in the
region and environmental variable. For unbalanced designs when a binary
environmental exposure has a low frequency in one category, GESAT is
most advantageous over single marker regression GxE test. Such
unbalanced designs can occur due to case--control sampling and the
strong association of an environmental factor with disease. When the
effect size is modest, GESAT performs better that single marker
regression GxE test, but when the effect size is strong, the opposite is
true. Their simulations suggest that the power of GESAT seems fairly
robust to the dependence between G and E.

The same approach can be applied to investigating various other
biological problems. For example, we can test for the interactions
between gene expressions in a pathway or network and an environmental
variable by simply replacing G by gene expressions in a gene-set.

Existing methods for assessing common variants by environment
interactions such as Gene-Environment Set Association Test (GESAT) (Lin
et al., 2013) have several limitations when applied for rare variants.
GESAT estimates the main effects of the common variants by applying a L2
penalty on the genotypes scaled to unit variance; this assumes that the
main effects of the scaled genotypes are comparable in magnitudes, which
may not hold in the case of rare variants. GESAT also assumes that the
regression coefficients of the rare variants by environment interactions
are independent of each other, and suffers from power loss when most
rare variants in a gene interact with the environmental factor and the
interaction effects have the same direction.

Similar idea from GESAT was later extended and applied to analyse rare
variants. GESAT have several limitations when applied for rare variants.
GESAT estimates the main effects of the common variants by applying a L2
penalty on the genotypes scaled to unit variance; this assumes that the
main effects of the scaled genotypes are comparable in magnitudes, which
may not hold in the case of rare variants. GESAT also assumes that the
regression coefficients of the rare variants by environment interactions
are independent of each other, and suffers from power loss when most
rare variants in a gene interact with the environmental factor and the
interaction effects have the same direction. The new proposed test iSKAT
is optimal in a class of variance component tests and is powerful and
robust to the proportion of variants in a gene that interact with
environment and the signs of the effects. This test properly controls
for the main effects of the rare variants using weighted ridge
regression while adjusting for covariates.

A naive approach to assess rare variants by environment interactions is
to extend the burden test by fitting a model with both the summary
genetic burden variable, environment, and their interaction, and
performing a one degree of freedom test for the interaction. However,
when there are multiple causal variants with their main effects having
different magnitudes and/or signs, such a burden rare variant by
environment test fails, and may lead to inflated Type 1 error rates.
This is because adjusting for the main effects of the multiple causal
variants using a single summary genetic burden variable is
inappropriate. Likewise, a naive approach to assess rare variants by
environment interactions using SKAT by including the main effects of
rare variants as part of covariates and applying SKAT to the interaction
terms is problematic. This is because SKAT only allows adjustment of a
small number of covariates and cannot handle the presence of a large
number of rare variants in a region. Furthermore since the rare variants
are observed in low frequency, a model with all the rare variants as
main effects will be highly unstable and may not even converge.

Both GESAT and iSKAT are able to incorporate multiple environmental
variables.

No gene-based GxE test exists for the analysis of the sex chromosomes as
of writing this text.

\section{Combining multiple
environments}\label{combining-multiple-environments}

\section{Variance heterogeneity}\label{variance-heterogeneity}

\subsection{Levene's test}\label{levenes-test}

\subsection{Two-step screening on residual variance
heterogeneity}\label{two-step-screening-on-residual-variance-heterogeneity}

\section{Conditional quantile
regression}\label{conditional-quantile-regression}

\section{RELIEF and other machine learning
tools}\label{relief-and-other-machine-learning-tools}

\subsection{Multidimensionality
reduction}\label{multidimensionality-reduction}

\section{Meta-analytic GxE
approaches}\label{meta-analytic-gxe-approaches}

\chapter{Gene-gene interaction}\label{gene-gene-interaction}

\section{Single step methods}\label{single-step-methods-1}

\section{Multi stage methods}\label{multi-stage-methods-1}

\section{Machine learning methods}\label{machine-learning-methods-1}

\chapter{Other omics}\label{other-omics}

\section{Transcriptome-wide association
studies}\label{transcriptome-wide-association-studies}

\subsection{cis eQTLs}\label{cis-eqtls}

\subsection{trans eQTLs}\label{trans-eqtls}

\subsection{3-D structure of the
genome}\label{d-structure-of-the-genome}

\section{Phenome-wide association
studies}\label{phenome-wide-association-studies}

\section{Metabolomics}\label{metabolomics}

\section{Epigenomics}\label{epigenomics}

\chapter{Quantitative trait loci
mapping}\label{quantitative-trait-loci-mapping}

\chapter{Additional points to
consider}\label{additional-points-to-consider}

\section{Kinship matrix}\label{kinship-matrix}

\subsection{Path coefficients}\label{path-coefficients}

\section{Genetic relationship matrix}\label{genetic-relationship-matrix}

\section{Animal models}\label{animal-models}

\section{Phasing}\label{phasing}

\section{Haplotyping}\label{haplotyping}

\section{Statistical power}\label{statistical-power}

\subsection{When marker is a disease susceptibility
locus}\label{when-marker-is-a-disease-susceptibility-locus}

\subsection{When marker is not disease susceptability
locus}\label{when-marker-is-not-disease-susceptability-locus}

\section{Multiple comparisons}\label{multiple-comparisons}

\subsection{Effective number of independant
variants}\label{effective-number-of-independant-variants}

\section{Biases}\label{biases}

\subsection{Ascertainment}\label{ascertainment}

\subsection{Attenuation}\label{attenuation}

\subsection{Selection}\label{selection}

\section{Family studies}\label{family-studies}

\subsection{Transmission disequilibrium
tests}\label{transmission-disequilibrium-tests}

\section{Twin studies}\label{twin-studies-1}

\section{Adoption studies}\label{adoption-studies}

\section{Equifinality (many genes give same
trait)}\label{equifinality-many-genes-give-same-trait}

\section{Gene dosage}\label{gene-dosage}

\subsection{Allelic dosage}\label{allelic-dosage}

\section{Allelic heterogeneity}\label{allelic-heterogeneity}

\section{Genetic heterogeneity}\label{genetic-heterogeneity}

\section{Genomic imprinting}\label{genomic-imprinting}

\section{Penetrance/phenocopy}\label{penetrancephenocopy}

\section{Endophenotypes}\label{endophenotypes}

\section{Ploidy}\label{ploidy}

\section{Exended phenotype}\label{exended-phenotype}

\section{Genome sizes}\label{genome-sizes}

\part{Population Genetics}\label{part-population-genetics}

\chapter{Genetic drift}\label{genetic-drift}

\chapter{Mutation}\label{mutation}

\section{Mutation age}\label{mutation-age}

\chapter{Selection}\label{selection-1}

\section{Directional}\label{directional}

\section{Balancing}\label{balancing}

\subsection{Frequency-dependent selection
1}\label{frequency-dependent-selection-1}

\subsection{Frequency dependent selection
2}\label{frequency-dependent-selection-2}

\section{Background selection}\label{background-selection}

\chapter{Migration}\label{migration}

\chapter{Diversity}\label{diversity}

\chapter{Admixture}\label{admixture}

\chapter{Linkage disequilibrium}\label{linkage-disequilibrium}

\chapter{In breeding and heterosis}\label{in-breeding-and-heterosis}

\chapter{Assortative mating}\label{assortative-mating}

\chapter{Identity}\label{identity}

\section{IBS}\label{ibs}

\subsection{Long runs of IBT}\label{long-runs-of-ibt}

\section{IBD}\label{ibd}

\section{IBT}\label{ibt}

\chapter{Neutral theory of molecular
evolution}\label{neutral-theory-of-molecular-evolution}

\section{Nearly neutral theory of molecular
evolution}\label{nearly-neutral-theory-of-molecular-evolution}


\end{document}
