<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>A handbook for Computational Genetics</title>
  <meta name="description" content="This is the book describing theoretical and practical approaches in analysis of genome data.">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="A handbook for Computational Genetics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is the book describing theoretical and practical approaches in analysis of genome data." />
  <meta name="github-repo" content="rstudio/bookdown-dem" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="A handbook for Computational Genetics" />
  
  <meta name="twitter:description" content="This is the book describing theoretical and practical approaches in analysis of genome data." />
  

<meta name="author" content="Alfred Pozarickij">


<meta name="date" content="2018-07-28">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="genetic-drift.html">
<link rel="next" href="selection-1.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="part"><span><b>I Quantitative Genetics</b></span></li>
<li class="chapter" data-level="1" data-path="population-parameters.html"><a href="population-parameters.html"><i class="fa fa-check"></i><b>1</b> Population parameters</a><ul>
<li class="chapter" data-level="1.1" data-path="population-parameters.html"><a href="population-parameters.html#mean"><i class="fa fa-check"></i><b>1.1</b> Mean</a></li>
<li class="chapter" data-level="1.2" data-path="population-parameters.html"><a href="population-parameters.html#variance"><i class="fa fa-check"></i><b>1.2</b> Variance</a></li>
<li class="chapter" data-level="1.3" data-path="population-parameters.html"><a href="population-parameters.html#covariance"><i class="fa fa-check"></i><b>1.3</b> Covariance</a></li>
<li class="chapter" data-level="1.4" data-path="population-parameters.html"><a href="population-parameters.html#genetic-correlation"><i class="fa fa-check"></i><b>1.4</b> Genetic correlation</a></li>
<li class="chapter" data-level="1.5" data-path="population-parameters.html"><a href="population-parameters.html#additivity"><i class="fa fa-check"></i><b>1.5</b> Additivity</a></li>
<li class="chapter" data-level="1.6" data-path="population-parameters.html"><a href="population-parameters.html#dominancerecesivness"><i class="fa fa-check"></i><b>1.6</b> Dominance/Recesivness</a></li>
<li class="chapter" data-level="1.7" data-path="population-parameters.html"><a href="population-parameters.html#codominance"><i class="fa fa-check"></i><b>1.7</b> Codominance</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="sequencing-technologies.html"><a href="sequencing-technologies.html"><i class="fa fa-check"></i><b>2</b> Sequencing technologies</a></li>
<li class="chapter" data-level="3" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html"><i class="fa fa-check"></i><b>3</b> Genome-wide association analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#genotype-calling-algorithms"><i class="fa fa-check"></i><b>3.1</b> Genotype calling algorithms</a></li>
<li class="chapter" data-level="3.2" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#dna-processing-quality-control"><i class="fa fa-check"></i><b>3.2</b> DNA processing quality control</a></li>
<li class="chapter" data-level="3.3" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#sample-quality-control"><i class="fa fa-check"></i><b>3.3</b> Sample quality control</a><ul>
<li class="chapter" data-level="3.3.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#cryptic-relatedness"><i class="fa fa-check"></i><b>3.3.1</b> Cryptic relatedness</a></li>
<li class="chapter" data-level="3.3.2" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#population-stratification"><i class="fa fa-check"></i><b>3.3.2</b> Population stratification</a></li>
<li class="chapter" data-level="3.3.3" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#heterozygosity-and-missingness-outliers"><i class="fa fa-check"></i><b>3.3.3</b> Heterozygosity and missingness outliers</a></li>
<li class="chapter" data-level="3.3.4" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#differential-missingness"><i class="fa fa-check"></i><b>3.3.4</b> Differential missingness</a></li>
<li class="chapter" data-level="3.3.5" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#sex-chromosome-anomalies"><i class="fa fa-check"></i><b>3.3.5</b> Sex chromosome anomalies</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#marker-quality-control"><i class="fa fa-check"></i><b>3.4</b> Marker quality control</a><ul>
<li class="chapter" data-level="3.4.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#genotyping-concordance"><i class="fa fa-check"></i><b>3.4.1</b> Genotyping concordance</a></li>
<li class="chapter" data-level="3.4.2" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#switch-rate"><i class="fa fa-check"></i><b>3.4.2</b> Switch rate</a></li>
<li class="chapter" data-level="3.4.3" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#genotype-call-rate"><i class="fa fa-check"></i><b>3.4.3</b> Genotype call rate</a></li>
<li class="chapter" data-level="3.4.4" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#minor-allele-frequency"><i class="fa fa-check"></i><b>3.4.4</b> Minor allele frequency</a></li>
<li class="chapter" data-level="3.4.5" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#hardy-weinberg-equilibrium-outliers"><i class="fa fa-check"></i><b>3.4.5</b> Hardy-Weinberg equilibrium outliers</a></li>
<li class="chapter" data-level="3.4.6" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#additional-qc-for-regions-like-mhc"><i class="fa fa-check"></i><b>3.4.6</b> Additional QC for regions like MHC</a></li>
<li class="chapter" data-level="3.4.7" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#ambigious-nucleotides"><i class="fa fa-check"></i><b>3.4.7</b> Ambigious nucleotides</a></li>
<li class="chapter" data-level="3.4.8" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#non-matching-nucleotides"><i class="fa fa-check"></i><b>3.4.8</b> Non-matching nucleotides</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#x-chromosome-quality-control"><i class="fa fa-check"></i><b>3.5</b> X-chromosome quality control</a></li>
<li class="chapter" data-level="3.6" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#single-marker-regression"><i class="fa fa-check"></i><b>3.6</b> Single marker regression</a><ul>
<li class="chapter" data-level="3.6.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#trend-test"><i class="fa fa-check"></i><b>3.6.1</b> Trend test</a></li>
<li class="chapter" data-level="3.6.2" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#alleles-test"><i class="fa fa-check"></i><b>3.6.2</b> Alleles test</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#two-stage-approach"><i class="fa fa-check"></i><b>3.7</b> Two-stage approach</a></li>
<li class="chapter" data-level="3.8" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#haplotype-gwas-design"><i class="fa fa-check"></i><b>3.8</b> Haplotype GWAs design</a><ul>
<li class="chapter" data-level="3.8.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#genomic-control"><i class="fa fa-check"></i><b>3.8.1</b> Genomic control</a></li>
</ul></li>
<li class="chapter" data-level="3.9" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#multimarker-gene-set-based-approaches"><i class="fa fa-check"></i><b>3.9</b> Multimarker gene-set based approaches</a></li>
<li class="chapter" data-level="3.10" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#extensions-to-binary-and-categorical-phenotypes"><i class="fa fa-check"></i><b>3.10</b> Extensions to binary and categorical phenotypes</a><ul>
<li class="chapter" data-level="3.10.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#threshold-model"><i class="fa fa-check"></i><b>3.10.1</b> Threshold model</a></li>
</ul></li>
<li class="chapter" data-level="3.11" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#analysis-of-rare-variants"><i class="fa fa-check"></i><b>3.11</b> Analysis of rare variants</a><ul>
<li class="chapter" data-level="3.11.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#collapsing-methods-based-on-pooling-multiple-rare-variants"><i class="fa fa-check"></i><b>3.11.1</b> Collapsing methods based on pooling multiple rare variants</a></li>
<li class="chapter" data-level="3.11.2" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#methods-treating-rare-variant-effects-as-random"><i class="fa fa-check"></i><b>3.11.2</b> Methods treating rare variant effects as random</a></li>
<li class="chapter" data-level="3.11.3" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#methods-based-on-model-selection"><i class="fa fa-check"></i><b>3.11.3</b> Methods based on model selection</a></li>
<li class="chapter" data-level="3.11.4" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#combination-of-collapsing-and-random-effects-methods"><i class="fa fa-check"></i><b>3.11.4</b> Combination of collapsing and random effects methods</a></li>
</ul></li>
<li class="chapter" data-level="3.12" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#analysis-of-x-y-and-mitochondrial-chromosomes"><i class="fa fa-check"></i><b>3.12</b> Analysis of X, Y and mitochondrial chromosomes</a></li>
<li class="chapter" data-level="3.13" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#analysis-of-copy-number-variants"><i class="fa fa-check"></i><b>3.13</b> Analysis of copy number variants</a><ul>
<li class="chapter" data-level="3.13.1" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#common-variation"><i class="fa fa-check"></i><b>3.13.1</b> Common variation</a></li>
<li class="chapter" data-level="3.13.2" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#analysis-of-rare-variants-1"><i class="fa fa-check"></i><b>3.13.2</b> Analysis of rare variants</a></li>
</ul></li>
<li class="chapter" data-level="3.14" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#analysis-of-multi-ethnic-samples"><i class="fa fa-check"></i><b>3.14</b> Analysis of multi-ethnic samples</a></li>
<li class="chapter" data-level="3.15" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#analysis-of-indirect-genetic-effects"><i class="fa fa-check"></i><b>3.15</b> Analysis of indirect genetic effects</a></li>
<li class="chapter" data-level="3.16" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#gwas-vs-whole-genome-association"><i class="fa fa-check"></i><b>3.16</b> GWAS vs whole-genome association</a></li>
<li class="chapter" data-level="3.17" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#analysis-of-multiple-traits"><i class="fa fa-check"></i><b>3.17</b> Analysis of multiple traits</a></li>
<li class="chapter" data-level="3.18" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#mixed-model-association-analysis"><i class="fa fa-check"></i><b>3.18</b> Mixed-model association analysis</a></li>
<li class="chapter" data-level="3.19" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#penalized-regression-gwas"><i class="fa fa-check"></i><b>3.19</b> Penalized regression GWAS</a></li>
<li class="chapter" data-level="3.20" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#bayesian-gwas"><i class="fa fa-check"></i><b>3.20</b> Bayesian GWAS</a></li>
<li class="chapter" data-level="3.21" data-path="genome-wide-association-analysis.html"><a href="genome-wide-association-analysis.html#machine-learning-for-gwas"><i class="fa fa-check"></i><b>3.21</b> Machine learning for GWAS</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="heritability.html"><a href="heritability.html"><i class="fa fa-check"></i><b>4</b> Heritability</a><ul>
<li class="chapter" data-level="4.1" data-path="heritability.html"><a href="heritability.html#realized-heritability"><i class="fa fa-check"></i><b>4.1</b> Realized heritability</a><ul>
<li class="chapter" data-level="4.1.1" data-path="heritability.html"><a href="heritability.html#evolvability"><i class="fa fa-check"></i><b>4.1.1</b> Evolvability</a></li>
<li class="chapter" data-level="4.1.2" data-path="heritability.html"><a href="heritability.html#reliability"><i class="fa fa-check"></i><b>4.1.2</b> Reliability</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="heritability.html"><a href="heritability.html#twin-studies"><i class="fa fa-check"></i><b>4.2</b> Twin studies</a></li>
<li class="chapter" data-level="4.3" data-path="heritability.html"><a href="heritability.html#gcta"><i class="fa fa-check"></i><b>4.3</b> GCTA</a></li>
<li class="chapter" data-level="4.4" data-path="heritability.html"><a href="heritability.html#ld-score-regression"><i class="fa fa-check"></i><b>4.4</b> LD-score regression</a></li>
<li class="chapter" data-level="4.5" data-path="heritability.html"><a href="heritability.html#ldak"><i class="fa fa-check"></i><b>4.5</b> LDAK</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="genomic-prediction.html"><a href="genomic-prediction.html"><i class="fa fa-check"></i><b>5</b> Genomic prediction</a><ul>
<li class="chapter" data-level="5.1" data-path="genomic-prediction.html"><a href="genomic-prediction.html#polygenic-risk-scores"><i class="fa fa-check"></i><b>5.1</b> Polygenic risk scores</a></li>
<li class="chapter" data-level="5.2" data-path="genomic-prediction.html"><a href="genomic-prediction.html#gene-based-polygenic-score-polaris"><i class="fa fa-check"></i><b>5.2</b> Gene-based polygenic score (POLARIS)</a></li>
<li class="chapter" data-level="5.3" data-path="genomic-prediction.html"><a href="genomic-prediction.html#pathway-based-polygenic-risk-score"><i class="fa fa-check"></i><b>5.3</b> Pathway-based polygenic risk score</a></li>
<li class="chapter" data-level="5.4" data-path="genomic-prediction.html"><a href="genomic-prediction.html#ld-adjusted-prs"><i class="fa fa-check"></i><b>5.4</b> LD adjusted PRS</a><ul>
<li class="chapter" data-level="5.4.1" data-path="genomic-prediction.html"><a href="genomic-prediction.html#ldpred-with-functional-annotation"><i class="fa fa-check"></i><b>5.4.1</b> LDpred with functional annotation</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="genomic-prediction.html"><a href="genomic-prediction.html#annopred"><i class="fa fa-check"></i><b>5.5</b> Annopred</a></li>
<li class="chapter" data-level="5.6" data-path="genomic-prediction.html"><a href="genomic-prediction.html#pleiopred"><i class="fa fa-check"></i><b>5.6</b> Pleiopred</a></li>
<li class="chapter" data-level="5.7" data-path="genomic-prediction.html"><a href="genomic-prediction.html#blup"><i class="fa fa-check"></i><b>5.7</b> BLUP</a></li>
<li class="chapter" data-level="5.8" data-path="genomic-prediction.html"><a href="genomic-prediction.html#bayesian-zoo"><i class="fa fa-check"></i><b>5.8</b> Bayesian Zoo</a></li>
<li class="chapter" data-level="5.9" data-path="genomic-prediction.html"><a href="genomic-prediction.html#reproducing-kernel-hilbert-space"><i class="fa fa-check"></i><b>5.9</b> Reproducing kernel Hilbert space</a></li>
<li class="chapter" data-level="5.10" data-path="genomic-prediction.html"><a href="genomic-prediction.html#machine-learning-methods"><i class="fa fa-check"></i><b>5.10</b> Machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="pleiotropy.html"><a href="pleiotropy.html"><i class="fa fa-check"></i><b>6</b> Pleiotropy</a><ul>
<li class="chapter" data-level="6.1" data-path="pleiotropy.html"><a href="pleiotropy.html#direct"><i class="fa fa-check"></i><b>6.1</b> Direct</a></li>
<li class="chapter" data-level="6.2" data-path="pleiotropy.html"><a href="pleiotropy.html#indirect"><i class="fa fa-check"></i><b>6.2</b> Indirect</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="pathway-analysis.html"><a href="pathway-analysis.html"><i class="fa fa-check"></i><b>7</b> Pathway-analysis</a></li>
<li class="chapter" data-level="8" data-path="functional-annotation.html"><a href="functional-annotation.html"><i class="fa fa-check"></i><b>8</b> Functional annotation</a></li>
<li class="chapter" data-level="9" data-path="causal-inference.html"><a href="causal-inference.html"><i class="fa fa-check"></i><b>9</b> Causal inference</a><ul>
<li class="chapter" data-level="9.1" data-path="causal-inference.html"><a href="causal-inference.html#gene-knockout"><i class="fa fa-check"></i><b>9.1</b> Gene-knockout</a></li>
<li class="chapter" data-level="9.2" data-path="causal-inference.html"><a href="causal-inference.html#conditioning"><i class="fa fa-check"></i><b>9.2</b> Conditioning</a></li>
<li class="chapter" data-level="9.3" data-path="causal-inference.html"><a href="causal-inference.html#finemapping"><i class="fa fa-check"></i><b>9.3</b> Finemapping</a></li>
<li class="chapter" data-level="9.4" data-path="causal-inference.html"><a href="causal-inference.html#mendelian-randomization"><i class="fa fa-check"></i><b>9.4</b> Mendelian Randomization</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="combining-multiple-datasets.html"><a href="combining-multiple-datasets.html"><i class="fa fa-check"></i><b>10</b> Combining multiple datasets</a><ul>
<li class="chapter" data-level="10.1" data-path="combining-multiple-datasets.html"><a href="combining-multiple-datasets.html#meta-analysis"><i class="fa fa-check"></i><b>10.1</b> Meta-analysis</a></li>
<li class="chapter" data-level="10.2" data-path="combining-multiple-datasets.html"><a href="combining-multiple-datasets.html#mega-analysis"><i class="fa fa-check"></i><b>10.2</b> Mega-analysis</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html"><i class="fa fa-check"></i><b>11</b> Gene-environment interaction</a><ul>
<li class="chapter" data-level="11.1" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#single-step-methods"><i class="fa fa-check"></i><b>11.1</b> Single step methods</a><ul>
<li class="chapter" data-level="11.1.1" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#case-control"><i class="fa fa-check"></i><b>11.1.1</b> Case-control</a></li>
<li class="chapter" data-level="11.1.2" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#case-only"><i class="fa fa-check"></i><b>11.1.2</b> Case-only</a></li>
<li class="chapter" data-level="11.1.3" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#empirical-bayes-and-bayesian-model-averaging"><i class="fa fa-check"></i><b>11.1.3</b> Empirical Bayes and Bayesian Model Averaging</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#multi-stage-methods"><i class="fa fa-check"></i><b>11.2</b> Multi stage methods</a></li>
<li class="chapter" data-level="11.3" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#joint-tests"><i class="fa fa-check"></i><b>11.3</b> Joint tests</a></li>
<li class="chapter" data-level="11.4" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#set-based-interaction-tests"><i class="fa fa-check"></i><b>11.4</b> Set-based interaction tests</a></li>
<li class="chapter" data-level="11.5" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#combining-multiple-environments"><i class="fa fa-check"></i><b>11.5</b> Combining multiple environments</a></li>
<li class="chapter" data-level="11.6" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#variance-heterogeneity"><i class="fa fa-check"></i><b>11.6</b> Variance heterogeneity</a><ul>
<li class="chapter" data-level="11.6.1" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#levenes-test"><i class="fa fa-check"></i><b>11.6.1</b> Leveneâ€™s test</a></li>
<li class="chapter" data-level="11.6.2" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#two-step-screening-on-residual-variance-heterogeneity"><i class="fa fa-check"></i><b>11.6.2</b> Two-step screening on residual variance heterogeneity</a></li>
</ul></li>
<li class="chapter" data-level="11.7" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#conditional-quantile-regression"><i class="fa fa-check"></i><b>11.7</b> Conditional quantile regression</a></li>
<li class="chapter" data-level="11.8" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#relief-and-other-machine-learning-tools"><i class="fa fa-check"></i><b>11.8</b> RELIEF and other machine learning tools</a><ul>
<li class="chapter" data-level="11.8.1" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#multidimensionality-reduction"><i class="fa fa-check"></i><b>11.8.1</b> Multidimensionality reduction</a></li>
</ul></li>
<li class="chapter" data-level="11.9" data-path="gene-environment-interaction.html"><a href="gene-environment-interaction.html#meta-analytic-gxe-approaches"><i class="fa fa-check"></i><b>11.9</b> Meta-analytic GxE approaches</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="gene-gene-interaction.html"><a href="gene-gene-interaction.html"><i class="fa fa-check"></i><b>12</b> Gene-gene interaction</a><ul>
<li class="chapter" data-level="12.1" data-path="gene-gene-interaction.html"><a href="gene-gene-interaction.html#single-step-methods-1"><i class="fa fa-check"></i><b>12.1</b> Single step methods</a></li>
<li class="chapter" data-level="12.2" data-path="gene-gene-interaction.html"><a href="gene-gene-interaction.html#multi-stage-methods-1"><i class="fa fa-check"></i><b>12.2</b> Multi stage methods</a></li>
<li class="chapter" data-level="12.3" data-path="gene-gene-interaction.html"><a href="gene-gene-interaction.html#machine-learning-methods-1"><i class="fa fa-check"></i><b>12.3</b> Machine learning methods</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="other-omics.html"><a href="other-omics.html"><i class="fa fa-check"></i><b>13</b> Other omics</a><ul>
<li class="chapter" data-level="13.1" data-path="other-omics.html"><a href="other-omics.html#transcriptome-wide-association-studies"><i class="fa fa-check"></i><b>13.1</b> Transcriptome-wide association studies</a><ul>
<li class="chapter" data-level="13.1.1" data-path="other-omics.html"><a href="other-omics.html#cis-eqtls"><i class="fa fa-check"></i><b>13.1.1</b> cis eQTLs</a></li>
<li class="chapter" data-level="13.1.2" data-path="other-omics.html"><a href="other-omics.html#trans-eqtls"><i class="fa fa-check"></i><b>13.1.2</b> trans eQTLs</a></li>
<li class="chapter" data-level="13.1.3" data-path="other-omics.html"><a href="other-omics.html#d-structure-of-the-genome"><i class="fa fa-check"></i><b>13.1.3</b> 3-D structure of the genome</a></li>
</ul></li>
<li class="chapter" data-level="13.2" data-path="other-omics.html"><a href="other-omics.html#phenome-wide-association-studies"><i class="fa fa-check"></i><b>13.2</b> Phenome-wide association studies</a></li>
<li class="chapter" data-level="13.3" data-path="other-omics.html"><a href="other-omics.html#metabolomics"><i class="fa fa-check"></i><b>13.3</b> Metabolomics</a></li>
<li class="chapter" data-level="13.4" data-path="other-omics.html"><a href="other-omics.html#epigenomics"><i class="fa fa-check"></i><b>13.4</b> Epigenomics</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="quantitative-trait-loci-mapping.html"><a href="quantitative-trait-loci-mapping.html"><i class="fa fa-check"></i><b>14</b> Quantitative trait loci mapping</a></li>
<li class="chapter" data-level="15" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html"><i class="fa fa-check"></i><b>15</b> Additional points to consider</a><ul>
<li class="chapter" data-level="15.1" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#kinship-matrix"><i class="fa fa-check"></i><b>15.1</b> Kinship matrix</a><ul>
<li class="chapter" data-level="15.1.1" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#path-coefficients"><i class="fa fa-check"></i><b>15.1.1</b> Path coefficients</a></li>
</ul></li>
<li class="chapter" data-level="15.2" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#genetic-relationship-matrix"><i class="fa fa-check"></i><b>15.2</b> Genetic relationship matrix</a></li>
<li class="chapter" data-level="15.3" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#animal-models"><i class="fa fa-check"></i><b>15.3</b> Animal models</a></li>
<li class="chapter" data-level="15.4" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#phasing"><i class="fa fa-check"></i><b>15.4</b> Phasing</a></li>
<li class="chapter" data-level="15.5" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#haplotyping"><i class="fa fa-check"></i><b>15.5</b> Haplotyping</a></li>
<li class="chapter" data-level="15.6" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#statistical-power"><i class="fa fa-check"></i><b>15.6</b> Statistical power</a><ul>
<li class="chapter" data-level="15.6.1" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#when-marker-is-a-disease-susceptibility-locus"><i class="fa fa-check"></i><b>15.6.1</b> When marker is a disease susceptibility locus</a></li>
<li class="chapter" data-level="15.6.2" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#when-marker-is-not-disease-susceptability-locus"><i class="fa fa-check"></i><b>15.6.2</b> When marker is not disease susceptability locus</a></li>
</ul></li>
<li class="chapter" data-level="15.7" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#multiple-comparisons"><i class="fa fa-check"></i><b>15.7</b> Multiple comparisons</a><ul>
<li class="chapter" data-level="15.7.1" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#effective-number-of-independant-variants"><i class="fa fa-check"></i><b>15.7.1</b> Effective number of independant variants</a></li>
</ul></li>
<li class="chapter" data-level="15.8" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#biases"><i class="fa fa-check"></i><b>15.8</b> Biases</a><ul>
<li class="chapter" data-level="15.8.1" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#ascertainment"><i class="fa fa-check"></i><b>15.8.1</b> Ascertainment</a></li>
<li class="chapter" data-level="15.8.2" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#attenuation"><i class="fa fa-check"></i><b>15.8.2</b> Attenuation</a></li>
<li class="chapter" data-level="15.8.3" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#selection"><i class="fa fa-check"></i><b>15.8.3</b> Selection</a></li>
</ul></li>
<li class="chapter" data-level="15.9" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#family-studies"><i class="fa fa-check"></i><b>15.9</b> Family studies</a><ul>
<li class="chapter" data-level="15.9.1" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#transmission-disequilibrium-tests"><i class="fa fa-check"></i><b>15.9.1</b> Transmission disequilibrium tests</a></li>
</ul></li>
<li class="chapter" data-level="15.10" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#twin-studies-1"><i class="fa fa-check"></i><b>15.10</b> Twin studies</a></li>
<li class="chapter" data-level="15.11" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#adoption-studies"><i class="fa fa-check"></i><b>15.11</b> Adoption studies</a></li>
<li class="chapter" data-level="15.12" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#equifinality-many-genes-give-same-trait"><i class="fa fa-check"></i><b>15.12</b> Equifinality (many genes give same trait)</a></li>
<li class="chapter" data-level="15.13" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#gene-dosage"><i class="fa fa-check"></i><b>15.13</b> Gene dosage</a><ul>
<li class="chapter" data-level="15.13.1" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#allelic-dosage"><i class="fa fa-check"></i><b>15.13.1</b> Allelic dosage</a></li>
</ul></li>
<li class="chapter" data-level="15.14" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#allelic-heterogeneity"><i class="fa fa-check"></i><b>15.14</b> Allelic heterogeneity</a></li>
<li class="chapter" data-level="15.15" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#genetic-heterogeneity"><i class="fa fa-check"></i><b>15.15</b> Genetic heterogeneity</a></li>
<li class="chapter" data-level="15.16" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#genomic-imprinting"><i class="fa fa-check"></i><b>15.16</b> Genomic imprinting</a></li>
<li class="chapter" data-level="15.17" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#penetrancephenocopy"><i class="fa fa-check"></i><b>15.17</b> Penetrance/phenocopy</a></li>
<li class="chapter" data-level="15.18" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#endophenotypes"><i class="fa fa-check"></i><b>15.18</b> Endophenotypes</a></li>
<li class="chapter" data-level="15.19" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#ploidy"><i class="fa fa-check"></i><b>15.19</b> Ploidy</a></li>
<li class="chapter" data-level="15.20" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#exended-phenotype"><i class="fa fa-check"></i><b>15.20</b> Exended phenotype</a></li>
<li class="chapter" data-level="15.21" data-path="additional-points-to-consider.html"><a href="additional-points-to-consider.html#genome-sizes"><i class="fa fa-check"></i><b>15.21</b> Genome sizes</a></li>
</ul></li>
<li class="part"><span><b>II Population Genetics</b></span></li>
<li class="chapter" data-level="16" data-path="genetic-drift.html"><a href="genetic-drift.html"><i class="fa fa-check"></i><b>16</b> Genetic drift</a></li>
<li class="chapter" data-level="17" data-path="mutation.html"><a href="mutation.html"><i class="fa fa-check"></i><b>17</b> Mutation</a><ul>
<li class="chapter" data-level="17.1" data-path="mutation.html"><a href="mutation.html#mutation-age"><i class="fa fa-check"></i><b>17.1</b> Mutation age</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="selection-1.html"><a href="selection-1.html"><i class="fa fa-check"></i><b>18</b> Selection</a><ul>
<li class="chapter" data-level="18.1" data-path="selection-1.html"><a href="selection-1.html#directional"><i class="fa fa-check"></i><b>18.1</b> Directional</a></li>
<li class="chapter" data-level="18.2" data-path="selection-1.html"><a href="selection-1.html#balancing"><i class="fa fa-check"></i><b>18.2</b> Balancing</a><ul>
<li class="chapter" data-level="18.2.1" data-path="selection-1.html"><a href="selection-1.html#frequency-dependent-selection-1"><i class="fa fa-check"></i><b>18.2.1</b> Frequency-dependent selection 1</a></li>
<li class="chapter" data-level="18.2.2" data-path="selection-1.html"><a href="selection-1.html#frequency-dependent-selection-2"><i class="fa fa-check"></i><b>18.2.2</b> Frequency dependent selection 2</a></li>
</ul></li>
<li class="chapter" data-level="18.3" data-path="selection-1.html"><a href="selection-1.html#background-selection"><i class="fa fa-check"></i><b>18.3</b> Background selection</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="migration.html"><a href="migration.html"><i class="fa fa-check"></i><b>19</b> Migration</a></li>
<li class="chapter" data-level="20" data-path="diversity.html"><a href="diversity.html"><i class="fa fa-check"></i><b>20</b> Diversity</a></li>
<li class="chapter" data-level="21" data-path="admixture.html"><a href="admixture.html"><i class="fa fa-check"></i><b>21</b> Admixture</a></li>
<li class="chapter" data-level="22" data-path="linkage-disequilibrium.html"><a href="linkage-disequilibrium.html"><i class="fa fa-check"></i><b>22</b> Linkage disequilibrium</a></li>
<li class="chapter" data-level="23" data-path="in-breeding-and-heterosis.html"><a href="in-breeding-and-heterosis.html"><i class="fa fa-check"></i><b>23</b> In breeding and heterosis</a></li>
<li class="chapter" data-level="24" data-path="assortative-mating.html"><a href="assortative-mating.html"><i class="fa fa-check"></i><b>24</b> Assortative mating</a></li>
<li class="chapter" data-level="25" data-path="identity.html"><a href="identity.html"><i class="fa fa-check"></i><b>25</b> Identity</a><ul>
<li class="chapter" data-level="25.1" data-path="identity.html"><a href="identity.html#ibs"><i class="fa fa-check"></i><b>25.1</b> IBS</a><ul>
<li class="chapter" data-level="25.1.1" data-path="identity.html"><a href="identity.html#long-runs-of-ibt"><i class="fa fa-check"></i><b>25.1.1</b> Long runs of IBT</a></li>
</ul></li>
<li class="chapter" data-level="25.2" data-path="identity.html"><a href="identity.html#ibd"><i class="fa fa-check"></i><b>25.2</b> IBD</a></li>
<li class="chapter" data-level="25.3" data-path="identity.html"><a href="identity.html#ibt"><i class="fa fa-check"></i><b>25.3</b> IBT</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="neutral-theory-of-molecular-evolution.html"><a href="neutral-theory-of-molecular-evolution.html"><i class="fa fa-check"></i><b>26</b> Neutral theory of molecular evolution</a><ul>
<li class="chapter" data-level="26.1" data-path="neutral-theory-of-molecular-evolution.html"><a href="neutral-theory-of-molecular-evolution.html#nearly-neutral-theory-of-molecular-evolution"><i class="fa fa-check"></i><b>26.1</b> Nearly neutral theory of molecular evolution</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A handbook for Computational Genetics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mutation" class="section level1">
<h1><span class="header-section-number">Chapter 17</span> Mutation</h1>
<div id="mutation-age" class="section level2">
<h2><span class="header-section-number">17.1</span> Mutation age</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="genetic-drift.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="selection-1.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/17-Mutation.Rmd",
"text": "Edit"
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
